tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -35,6 +35,10 @@ const (
 	AppendOps Ops = iota
 	// DeleteOps performs delete operations of rules
 	DeleteOps
+
+	// In TPROXY mode, mark the packet from envoy outbound to app by podIP,
+	// this is to make it later intercepted to envoy inbound listener.
```

Comment:
`this is to make it later intercepted to envoy inbound listener.`
The comment doesn't reflect the diagram in https://drive.google.com/file/d/1A6SLvwiRcEUp8WEHEQfjiDQpylhiSP5O/view?usp=sharing 

`prevent intercept to envoy inbound`?

Reply:
right, will update comment